### PR TITLE
docs([DST-485]): Create reusable component for Do's and Don'ts

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**

### CODE REVIEW

File: docs/ui/Do.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import { Image } from '@/ui/Image';
+
+interface Props {
+  img: string;
+  description: string;
+}
+
+export const Do = ({ description, img }: Props) => {
+  return (
+    <div className="border-border flex flex-col border">
```
Comment (sebald) [inline docs/ui/Do.tsx:12]: use a card as a rapper? this would fit niceley with all the other stuff we have :)

**Comment by OsamaAbdellateef** [comment]
Also what do you think to have two components Do and Dont 
I could make them into one component but in terms of code it didn't look great and a lot of conditions @sebald  @sarahgm

File: docs/ui/Do.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <Image width={700} height={900} src={img} alt="best practice" />
```
Comment (sebald) [inline docs/ui/Do.tsx:13]: Maybe this can be a `children` so we can use something different here if we want to

File: docs/ui/Dont.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { Text } from '@marigold/components';
```
Comment (sebald) [inline docs/ui/Dont.tsx:1]: I think we can put them in one file, makes it easier to find them and we need to adjust both anyway when we need to change something.

**Comment by sebald** [comment]
> Also what do you think to have two components Do and Dont 
> I could make them into one component but in terms of code it didn't look great and a lot of conditions @sebald  @sarahgm 

Yes that would be awesome

File: docs/ui/AdviceCard.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import React, { ReactNode } from 'react';
+
+import { Card, Headline, Text } from '@marigold/components';
+
+import { cn } from '../../packages/system/dist';
+
+interface Props {
+  description: string;
+  type: 'DO' | "DON'T";
```
Comment (sarahgm) [inline docs/ui/AdviceCard.tsx:9]: I think it's a bad idea to write it uppercase and with a ' because its more error-prone, I would think

Comment (sarahgm) [reply docs/ui/AdviceCard.tsx:9]: and you can also change type to variant?

**Comment by sarahgm** [comment]
how do we create actual do and don'ts? I mean the content of it

File: docs/ui/Do.tsx (hunk 1 above)
Comment (OsamaAbdellateef) [reply docs/ui/Do.tsx]: Now there is discussion about the Card to use it or not 
https://marigold-docs-git-do-card-marigold.vercel.app/components/content/card?vercelThreadId=kuiD6
@sebald

File: .changeset/thin-suits-complain.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
+'@marigold/docs': patch
```
Comment (sebald) [inline .changeset/thin-suits-complain.md:2]:
```suggestion
'@marigold/docs': minor
```

**Review by sebald — APPROVED** [review]

**Pull request merged**